azext_edge/edge/providers/rpsaas/adr/helpers.py

```diff
@@ -23,6 +30,16 @@ def get_extended_location(
     instance_resource_group: str,
     instance_subscription: Optional[str] = None
 ) -> Dict[str, str]:
+    """
+    Returns the extended location object with cluster location.
+
+    Will also check for instance existance and whether the associated cluster is connected.
+
+    instance_name: str representing the instance name
+    instance_resource_group: str representing the instance resource group
+    instance_instance_subscriptionname: str representing the instance subscription
```

Comment:
instance_instance_subscriptionname

Reply:
kk will change it to:
`instance_instance_instance_subscription_instance`

jk